scripts/install.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -euo pipefail

```

Should we enable strict mode to avoid potential mess later down the line?

See: http://redsymbol.net/articles/unofficial-bash-strict-mode/

Reply:
Done ;)